src/libcmd/installables.cc

```diff
@@ -505,6 +505,26 @@ struct InstallableAttrPath : InstallableValue
 
         return res;
     }
+
+    std::vector<ref<eval_cache::AttrCursor>> getCursors(EvalState &state) override
```

Comment:
I don't know if the getCursors change will impact other commands, i suspect it would now do an autoCall where before it didn't. It might be desirable to do so, but it is not clear how much behavior this is overriding.